### ISSUE TITLE
Pull all HBAI data split by every dimension

### ISSUE THREAD

**Issue by luke-strange** [open-innovations/jrf-insight#12]
Pull the entire HBAI database - for each measure, have a table with every dimension. 

For now we just want to pull the most recent year otherwise we get blocked by the limit of 1,000,000 cells.

**Comment by luke-strange** [comment]
BLOCKED by #13

**Comment by luke-strange** [comment]
statXplore has a limit of 1,000,000 cells, so we will probably have to pull this data 1 year at a time.

**Comment by luke-strange** [comment]
Also want to pull last time series element for everything to get an idea of what is in the data.

**Comment by luke-strange** [comment]
Blocked by waiting for #39

**Comment by luke-strange** [comment]
well, it looks like we won't be able to pull every dimension, even when we limit to 1 year at a time. We will need to decide which dimensions we can leave out, there are probably some that won't be particularly useful for us. @ChristianSpence

**Comment by ChristianSpence** [comment]
Is this a volume of data to pull issue, or something else?

**Comment by luke-strange** [comment]
@ChristianSpence i believe its volume yes.

**Comment by ChristianSpence** [comment]
OK, let's discuss after the sprint planning.

**Comment by luke-strange** [comment]
clsoing this as we have the data we need as stands. we can create a new ticket and review statxplore data pulling if/when necessary.
